.github/workflows/cicd.yml

```
@@ -7,7 +7,48 @@ on:
 
   push:
     branches:
+<<<<<<< HEAD
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 10: could not find expected ':'

(syntax-check)